### PR TITLE
Contribute .md and testing instructions

### PR DESCRIPTION
**Pull request by rihp**
### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
-# Contributing to Auto-GPT
+# Contributing to ProjectName
```
Comment (Pwuts) [inline CONTRIBUTING.md]: ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-- [Code of Conduct](#code-of-conduct)
-- [Getting Started](#getting-started)
-- [How to Contribute](#how-to-contribute)
-  - [Reporting Bugs](#reporting-bugs)
-  - [Suggesting Enhancements](#suggesting-enhancements)
-  - [Submitting Pull Requests](#submitting-pull-requests)
-- [Style Guidelines](#style-guidelines)
-  - [Code Formatting](#code-formatting)
-  - [Pre-Commit Hooks](#pre-commit-hooks)
+- [Contributing to ProjectName](#contributing-to-projectname)
```
Comment (Pwuts) [inline CONTRIBUTING.md]: ProjectName?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -102,4 +106,14 @@ Happy coding, and once again, thank you for your contributions!
...
-https://github.com/Significant-Gravitas/Auto-GPT/pulls?q=is%3Apr+is%3Aopen+-is%3Aconflict+
+https://github.com/Torantulino/Auto-GPT/pulls?q=is%3Apr+is%3Aopen+-is%3Aconflict+
+
+## Running tests
+
+Execute the test suite by calling `pytest` from your CLI
+
+You can specify which tests you want to run by calling `pytest tests/filename.py`
+
+Use pytest over unit test for creating tests.
```
Comment (Pwuts) [inline CONTRIBUTING.md]:
```suggestion
Use `pytest` over `unittest` for creating tests.
```

**Review by Pwuts — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -102,4 +106,14 @@ Happy coding, and once again, thank you for your contributions!
 
 Maintainers will look at PR that have no merge conflicts when deciding what to add to the project. Make sure your PR shows up here:
 
-https://github.com/Significant-Gravitas/Auto-GPT/pulls?q=is%3Apr+is%3Aopen+-is%3Aconflict+
+https://github.com/Torantulino/Auto-GPT/pulls?q=is%3Apr+is%3Aopen+-is%3Aconflict+
```
Comment (hdkiller) [inline CONTRIBUTING.md]: Why link the old name instead of the new one?

File: CONTRIBUTING.md (hunk 2 above)
Comment (rihp) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (rihp) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 1 above)
Comment (rihp) [reply CONTRIBUTING.md]: Fixed

**Pull request closed without merging**